mods/default/mapgen.lua

```diff
@@ -1986,6 +1986,28 @@ function default.register_decorations()
 		flags = "place_center_x, place_center_z",
 	})
 
+	-- Pine bush
+
+	minetest.register_decoration({
+		name = "default:pine_bush",
+		deco_type = "schematic",
+		place_on = {"default:dirt_with_snow"},
+        sidelen = 16,
```

Comment:
Weird indent

Reply:
sorry, that is weird.. it was an indent of spaces rather than tabs.
fixed now